### PR TITLE
Add _msearch elasticsearch endpoint.

### PR DESCRIPTION
**Pull request by fmassot**

### CODE REVIEW

**Comment by fmassot** [comment]
I should have approved @trinity-1686a  's PR once this one was merged! what a mistake ;)

**Comment by trinity-1686a** [comment]
do you want to revert #3342 so this one can go through more rapidly?

**Comment by fmassot** [comment]
thanks @trinity-1686a  but I will handle that, no pb.

File: quickwit/quickwit-serve/src/elastic_search_api/model/multi_search.rs
Comment (fulmicoton) [inline quickwit/quickwit-serve/src/elastic_search_api/model/multi_search.rs:32]: It might be worth adding a 

#[serde(deny_unknown_fields)]

File: quickwit/quickwit-serve/src/elastic_search_api/model/multi_search.rs
Comment (fulmicoton) [inline quickwit/quickwit-serve/src/elastic_search_api/model/multi_search.rs:65]: link to the ES doc maybe?

**Review by fulmicoton — APPROVED** [review]

**Pull request merged**
